docs/contributing/documentation_guidelines.md

```diff
@@ -177,13 +213,13 @@ A [link to a reference](register_reports_menu_item).
 An [external link](https://wwww.example.com).
 An [internal link to another document](/reference/contrib/legacy_richtext).
 An auto generated link label to a page [](/getting_started/tutorial).
-A [link to a reference](register_reports_menu_item).
+A [link to a target](register_reports_menu_item).
 
 </details>
 
-#### Reference links
+#### Anchor links
 
-Reference links (links to a target within a page) rely on the page having a reference created. Each reference must have a unique name and should use the `lower_snake_case` format. A reference can be added as follows:
```

Comment:
I changed the terminology from "reference" to "anchor" here because it's a bit ambiguous with "reference" as in "code/API reference"